assets/js/mainscript.js

```diff
@@ -0,0 +1,59 @@
+const speakersArray = [
+  {
+    name: 'Alexander',
+    position: 'Software Engineer & Video Game Programmer',
+    description: '5 years of expericence developing games and responsible for shipping Triple A titles, which are hopefully not false claims ironically',
+    imageLink: 'assets/images/Alex.jpeg',
+  },
+  {
+    name: 'Khalid',
+    position: 'Software Engineer & Data Analyst Expert',
+    description: '5 years of expericence of big data analysis and data matching of okcupid',
+    imageLink: 'assets/images/khalid.jpg',
+  },
+  {
+    name: 'James',
+    position: '',
+    description: 'Exploring the future of media in Facebooks first Virtual Reality app a place to discover and enjoy 360 photos and videos on Gear VR',
+    imageLink: 'assets/images/james.jpg',
+  },
+  {
+    name: 'Hunter',
+    position: 'Software Engineer',
+    description: 'Software Engineer at Nvidia corps, chief architect engineer responsible for DLSS algorithms in Nvidia graphics cards',
+    imageLink: 'assets/images/hunter.jpg',
+  },
+];
+
```

Comment:
- [ ] **Required:** Kindly duplicate the speakers to be a total of 6 in this section below:
![image](https://user-images.githubusercontent.com/27709832/152191940-302915b8-6600-4c23-a665-d57637850c7e.png)
![image](https://user-images.githubusercontent.com/27709832/152192067-018b75ea-2ff7-430e-a959-56d7ec916b50.png)